client/components/deposits-overview/utils.ts

```diff
@@ -39,7 +39,7 @@ export const getNextDeposit = (
 	return {
 		id: nextScheduled.id,
 		date: nextScheduled.date ?? 0,
-		status: nextScheduled.status ?? 'estimated',
+		status: nextScheduled.status ?? '-',
 		amount: formatCurrency( nextScheduled.amount ?? 0, currency ),
```

Comment:
I'm a little confused by these changes – `getNextDeposit()` returns `status: -`, what does this mean? Is the code dead? If it returns `estimated` what breaks?

I guess this code is becoming redundant; `getNextDeposit` seems coupled to the idea of estimating/predicting deposits.